index.html

```diff
@@ -152,7 +152,9 @@
 
     <p>
 <a>Decentralized identifiers</a> (DIDs) are a new type of identifier to
-provide verifiable, decentralized digital identity. These new identifiers are
+enable verifiable, decentralized digital identity. A DID identifies an 
+entity that is capable to digitally interact, e.g. a natural person, 
```

Comment:
```suggestion
subject, e.g. a natural person, 
```

Not sure if this is helpful, but I would love the term "did subject" to be mentioned at this point in the document

Reply:
I have no problem to replace "entity" with "subject". However, I don't want to remove "that is capable to digitally interact", because that is the essence of of the PR. 

The whole purpose of obtaining keys and service endpoints associated with a DID subject is to interact with that DID subject, even though the DID controller happens to be different from the DID subject. E.g. if the DID subject is "IBM", then the DID controller may be an IT system that is managed by some IT guys at IBM who follow their management's policies.